numpy/linalg/umath_linalg.c.src

```diff
@@ -3235,6 +3252,289 @@ static void
 
 /**end repeat**/
 
+/* -------------------------------------------------------------------------- */
+                 /* qr (modes - r, raw, economic) */
```

Comment:
I don't think you should mention `economic` here as it's just a deprecated spelling of `raw` with some weirdness that C doesn't care about.

Reply:
Okay. I will remove it. The C implementation will remain the same as `r`, `raw` and `economic` use the computation done by this function.